packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts

```diff
@@ -337,6 +343,12 @@ export interface IPersonaPresenceStyleProps {
    */
   presence?: PersonaPresence;
 
+  /**
+   * This flag can be used to signal the persona is out of office.
```

Comment:
I like that :D

Ideally I'd like to get rid of the `IPersonaSharedProps` type as imo it makes the whole thing a bit more confusing, and I would like all three components to define their own props rather than sharing a single interface that is a superset of the actual props.

Reply:
Yeah if not all the components even use all the "shared" props, that's definitely an argument for moving those props out of the interface or getting rid of the interface altogether. Unfortunately that would have to be done as part of a major version bump.